security/providers/oidc-common/src/main/java/io/helidon/security/providers/oidc/common/OidcConfig.java

```diff
@@ -1087,6 +1123,10 @@ public Builder config(Config config) {
             config.get("tenants").asList(Config.class)
                     .ifPresent(confList -> confList.forEach(tenantConfig -> tenantFromConfig(config, tenantConfig)));
 
+            config.get("webclient-config")
```

Comment:
`webclient-config` in config seems like a too much information.
Can we just use `webclient`?